doc/rbd/iscsi-target-cli-manual-install.rst

```diff
@@ -147,6 +145,7 @@ on each machine that will be a iSCSI gateway:
        > cd ceph-iscsi-cli
        > python setup.py install
        > cp usr/lib/systemd/system/rbd-target-api.service /lib/systemd/system
+       > cp rbd-target-api.py /usr/bin/rbd-target-api
```

Comment:
The ```setup.py``` script is supposed to strip the ```.py``` extension from the scripts [1][2]. Is it not working for some reason for you?

[1] https://github.com/ceph/ceph-iscsi-cli/blob/master/setup.py#L45
[2] https://github.com/ceph/ceph-iscsi-config/blob/master/setup.py#L46

Reply:
Hmm, it does. But it copies to "/usr/local/bin". The systemd unit file has a ExecStart path of "/usr/bin/rbd-target-api":

ExecStart=/usr/bin/rbd-target-api

I checked the status of the service: reporting it couldn't find the script. I assumed an extra "cp" step was needed, but it turns out the default install path, on Ubuntu 18.04 LTS at least, is /usr/local/bin.

Installing with "python setup.py install --install-scripts=/usr/bin" results in the script being installed in /usr/bin (without the .py extension)

Maybe add the "install --install-scripts=/usr/bin" to ensure where the script will be installed?

Reply:
Sure, thx